lib/config.js

```diff
@@ -14,6 +14,14 @@ const conf = convict({
       default: 1
     }
   },
+  authServer: {
+    url: {
+      doc: 'URL of fxa-auth-server',
+      env: 'AUTH_SERVER_URL',
+      format: 'url',
+      default: 'http://127.0.0.1:9000/v1'
+    }
+  },
```

Comment:
Style nit: we should put this next to the `oauth.url` config option and ensure they're named consistently.  Right now it's "oauth.url" and "authServer.url", and I don't strongly prefer one over the other, but let's pick a consistent style.